godot_project/python/scripts/openmm_server.py

```diff
@@ -19,6 +19,10 @@
 
 DETAILED_LOGS = False
 
+try:
+	os.mkdir(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "..", "logs" ))
```

Comment:
this one fixes some other issue?

Reply:
i got an exception when running from the debugger, so i added this guard 🤷